testing/dart/image_descriptor_test.dart

```diff
@@ -70,6 +70,19 @@ void main() {
     final Codec codec = await descriptor.instantiateCodec();
     expect(codec.frameCount, 1);
   });
+
+  test('HEIC image', () async {
+    final Uint8List bytes = await readFile('grill_chicken.heic');
```

Comment:
This is a photo I took on an iPhone and downscaled a bit.

Reply:
🍗